moped-database/migrations/1658526083849_insert_into_public_moped_components/up.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO "public"."moped_components"(
+	"component_name", "status_id", "component_subtype", "line_representation") 
+VALUES 
+	(E'Lighting', 1, E'Street lighting', true),
+	(E'Lighting', 1, E'Intersection lighting', false),
+	(E'Transit', 1, E'Bus stop', false);
```

Comment:
I always have to remind myself what Postgres Escape Notation `E''` does when I see it. It causes it to respect the usual suspects like `\n`, `\t` to get interpolated into the actual newline or tab character.